### ISSUE TITLE
error: package ... not found registry.npmjs.org/@... 404

### ISSUE THREAD

**Issue by nearestnabors** [oven-sh/bun#1187]
### Version

0.1.10

### Platform

Darwin 3c06304b92ef 21.6.0 Darwin Kernel Version 21.6.0: Sat Jun 18 17:05:47 PDT 2022; root:xnu-8020.140.41~1/RELEASE_ARM64_T8101 arm64

### What steps will reproduce the bug?

I was going through [the AWS Amplify tutorial for Next.js](https://docs.amplify.aws/start/getting-started/setup/q/integration/next/#initialize-a-new-backend) and replacing `npm` with `bun` all along the way. But when I got to `bun install aws-amplify @aws-amplify/ui-react` I got this:

```
$ bun install aws-amplify @aws-amplify/ui-react
bun add v0.1.10
  🔍 @aws-amplify/ui-react-v1 [69/210] 
error: package "@aws-amplify/ui-react-v1" not found registry.npmjs.org/@aws-amplify/ui-react-v1 404
  🚚 @radix-ui/react-slider [1475/1476] 
```

When I used `npm install aws-amplify @aws-amplify/ui-react` things installed just fine.

Will bun be supporting this syntax anytime soon?

### What do you see instead?

```
$ bun install aws-amplify @aws-amplify/ui-react
bun add v0.1.10
  🔍 @aws-amplify/ui-react-v1 [69/210] 
error: package "@aws-amplify/ui-react-v1" not found registry.npmjs.org/@aws-amplify/ui-react-v1 404
  🚚 @radix-ui/react-slider [1475/1476] 
```

**Comment by dbanksdesign** [comment]
Here is the PR on the npm CLI adding support for aliased dependencies: https://github.com/npm/cli/pull/3

**Comment by Jarred-Sumner** [comment]
Fixed by @alexlamsl in #1837
